yb_migrate/src/utils/commonVariables.go

```diff
@@ -94,3 +100,41 @@ type Issue struct {
 	Suggestion   string `json:"suggestion"`
 	GH           string `json:"GH"`
 }
+
+type DataFileDescriptor struct {
+	FileType      string           `json:"FileType"`
+	TableRowCount map[string]int64 `json:"TableRowCount"`
+	Delimiter     string           `json:"Delimiter"`
+	HasHeader     bool             `json:"HasHeader"`
+}
+
+func (dfd *DataFileDescriptor) StoreDataFileDescriptor(exportDir string) {
+	dfdFilePath := exportDir + "/metainfo/flags/dataFileDescriptor"
+	log.Infof("storing DataFileDescriptor at %q")
+
+	bytes, err := json.MarshalIndent(dfd, "", "\t")
+	if err != nil {
+		ErrExit("Marshalling the dfd struct: %v", err)
+	}
+
+	err = ioutil.WriteFile(dfdFilePath, bytes, 0644)
+	if err != nil {
+		ErrExit("writing DataFileDescriptor at %q: %v", dfdFilePath, err)
+	}
+}
+
+func (dfd *DataFileDescriptor) LoadDataFileDescriptor(exportDir string) {
+	if dfd != nil {
+		return // return if already loaded
+	}
+	dfdFilePath := exportDir + "/metainfo/flags/dataFileDescriptor"
+	log.Infof("loading DataFileDescriptor from %q", dfdFilePath)
+	dfdJson, err := ioutil.ReadFile(dfdFilePath)
+	if err != nil {
+		ErrExit("reading DataFileDescriptor data: %v", err)
+	}
+
+	json.Unmarshal(dfdJson, dfd)
+	fmt.Printf("DataFileDescriptor: %+v\n", dfd)
+	os.Exit(1)
```

Comment:
What's that!!!

Reply:
It was there in a initial commit, had removed it in later ones.